Formula/consul.rb

```diff
@@ -31,7 +31,8 @@ def install
     on_linux do
       ENV["XC_OS"] = "linux"
     end
-    ENV["XC_ARCH"] = "amd64"
+    cpu = Hardware::CPU.arm? ? "arm64" : "amd64"
+    ENV["XC_ARCH"] = cpu
```

Comment:
```suggestion
    ENV["XC_ARCH"] = Hardware::CPU.arm? ? "arm64" : "amd64"
```
Do we need two lines for this?